nf_core/pipeline-template/README.md

```diff
@@ -25,57 +25,76 @@
+<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
+     Explain what rows and columns represent.
+ -->
 
-3. Download the pipeline and test it on a minimal dataset with a single command:
+First, you need to prepare a samplesheet with your input data that looks as follows:
 
-   ```bash
-   nextflow run {{ name }} -profile test,YOURPROFILE --outdir <OUTDIR>
-   ```
+**samplesheet.csv**:
 
-   Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
+```csv
+sample,fastq_1,fastq_2
+CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
+```
 
-   > - The pipeline comes with config profiles called `docker`, `singularity`, `podman`, `shifter`, `charliecloud` and `conda` which instruct the pipeline to use the named tool for software management. For example, `-profile test,docker`.
-   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
-   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
-   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
+Each row represents a fastq file (single-end) or a pair of fastq files (paired end).
 
-4. Start running your own analysis!
+Now, you can run the pipeline using:
 
-   <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
+<!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
 
-   ```bash
-   nextflow run {{ name }} --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
-   ```
+```bash
+nextflow run {{ name }} \
+   --input samplesheet.csv \
+   -profile <docker/singularity/.../institute>
```

Comment:
Any reason to drop outdir and genome? Outdir is typically a required param in basically all pipelines, so I think it should be there.

Might also be good to put core options such as `-profile` first so as not to mix numbers of hyphens.

```suggestion
nextflow run {{ name }} \
   -profile <docker/singularity/.../institute> \
   --input samplesheet.csv \
   --genome GRCh37
   --outdir <OUTDIR>
```

Reply:
Agreed, genome is actually only specific to a small subset of pipelines (many will have `fasta` instead as they need to support non iGenomes referecnes; or have no reference at all)

Reply:
ok sure 👍🏻

Reply:
```suggestion
nextflow run {{ name }} \
   -profile <docker/singularity/.../institute> \
   --input samplesheet.csv \
   --outdir <OUTDIR>
```